preload.js

```diff
@@ -1,4 +1,7 @@
 const { remote, ipcRenderer } = require('electron');
+const Store = require('electron-store');
+
+const settings = new Store({ name: 'settings' });
```

Comment:
What's the purpose of this new store?

Reply:
Nevermind. I checked this issue and got it https://github.com/sindresorhus/electron-store/issues/109